snap/snapcraft.yaml

```diff
@@ -155,6 +126,7 @@ plugs:
   network-observe: null
   network-manager: null
   opengl: null
+  pipewire: null
```

Comment:
It might be easier to leave this out at least for the initial merge, since it is likely to fail automated review. Once we've got a published revision without pipewire services for testing, it can be added back.

Reply:
My fault. This should be commented and with a note explaining to uncomment it when the interface is ready.